cluster/manifests/e2e-resources/pool-reserve.yaml

```diff
@@ -1,5 +1,5 @@
 {{ if eq .Environment "e2e" }}
-{{ range $pool := split "default-worker-splitaz,default-worker,worker-limit-az,worker-instance-storage,default-worker-spotio,worker-node-tests" "," }}
+{{ range $pool := split "default-worker-splitaz,default-worker,worker-limit-az,worker-instance-storage,worker-node-tests" "," }}
```

Comment:
I think you'll need a deletion as well, otherwise the update would fail.

Reply:
Thanks! I added this and it's working now!